src/maheshKJun20/ArrayExample/AllIndexwithDifference.java

```diff
@@ -0,0 +1,37 @@
+/*
+ Find all differ index from given arrays. 
+i/p:- arr1: {10,2,9,14,3}
+arr2: {10,2,18,14,13}
+o/p:- Values are not matching at index -> 2
+Values are not matching at index -> 4
+*/
+
+package maheshKJun20.ArrayExample;
+
+public class AllIndexwithDifference {
+
+	void displayIndexWithDiffValue(int[] intArray1, int[] intArray2) {
+		//Method to display all index with difference in value of 2 interger Arrays
+		boolean flag = true;
+		if(intArray1.length == intArray2.length) {
+			for (int index=0; index<intArray1.length;index++) {
+				if (intArray1[index] != intArray2[index]) {
+					System.out.println("Values are not matching at index: " + index);
+					flag = false;
+				}				
+			}
+			if (flag)
```

Comment:
Same as here, boolean flag = true; this variable is not needed.

Reply:
Updated the Code as per above comments